.github/workflows/microservice.yml

```
@@ -121,6 +121,11 @@ jobs:
           ref: ${{ inputs.target_branch }}
         if: inputs.submodules == 'recursive' && github.event_name == 'workflow_dispatch'
 
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
```

```suggestion
          go-version: '1.21'
          check-latest: true
```

so the latest go 1.21 with CVE patches is used.